plugins/orchestrator-backend/src/service/SonataFlowService.ts

```diff
@@ -233,6 +233,12 @@ export class SonataFlowService {
     return undefined;
   }
 
+  public async fetchWorkflowOverview(
+    workflowId: string,
+  ): Promise<WorkflowOverview | undefined> {
+    return this.getWorkflowOverview(workflowId);
```

Comment:
Please consider renaming `getWorkflowOverview` to `fetchWorkflowOverview` and making it public. 
This way, you don't need a method that simply calls another method with the same args and no additional logic.
wdyt?

Reply:
[6de85ad](https://github.com/caponetto/backstage-plugins/pull/40/commits/6de85ad36dd92985c7a42b95ae9eb3c6cad32db7)